types/underscore/underscore-tests.ts

```diff
@@ -681,28 +681,70 @@ _.chain([1, 2, 3, 4, 5, 6])
     .value();
 
 // common testing types and objects
+const context = {};
+
 interface StringRecord {
     a: string;
     b: string;
 }
 
+const stringRecordProperty = 'a';
+const partialStringRecord: Partial<StringRecord> = { a: 'b' };
+
 interface StringRecordAugmentedList extends _.List<StringRecord> {
     notAListProperty: boolean;
 }
 
+const stringRecordArray: StringRecord[] = [{ a: 'a', b: 'c' }, { a: 'b', b: 'b' }, { a: 'c', b: 'a' }];
```

Comment:
Not to state the obvious, but expressions like these are really easy to break over multiple lines in a neat way.

Reply:
I compressed this so it would be less emphasized because the implementation was nice to have as an example but not really actually important. I could either spread this across multiple lines or drop the implementation altogether and use `declare` instead; do you have a preference?

Reply:
Ah of course. I think I prefer `declare`.